components/automate-ui/karma.conf.js

```diff
@@ -25,9 +25,15 @@ module.exports = function (config) {
     client:{
       clearContext: false // leave Jasmine Spec Runner output visible in browser
     },
-    coverageIstanbulReporter: {
-      dir: require('path').join(__dirname, 'coverage'), reports: [ 'html', 'lcovonly', 'json-summary' ],
-      fixWebpackSourcePaths: true
+
+    coverageReporter: {
+      dir: require('path').join(__dirname, 'coverage'),
+      subdir: '.',
+      reporters: [
+        { type: 'html' },
+        { type: 'lcovonly' },
+        { type: 'json-summary' }
+      ],
```

Comment:
this is mapping the same way as it did previously, just needed a differently named object and no longer needs `fixWebpackSourcePaths`.  Its no longer even listed as an option: https://karma-runner.github.io/6.2/config/configuration-file.html